lighthouse-core/report/v2/renderer/details-renderer.js

```diff
@@ -88,6 +116,33 @@ class DetailsRenderer {
     return element;
   }
 
+
+  /**
+   * @param {!DetailsRenderer.TableDetailsJSON} details
+   * @return {!Element}
+   */
+  _renderTable(details) {
+    const element = this._dom.createElement('details', 'lh-details', {open: true});
+    if (details.header) {
+      element.appendChild(this._dom.createElement('summary')).textContent = details.header;
+    }
+
+    const tableElem = element.createChild('table', 'lh-table lh-table__multicolumn');
```

Comment:
Do you need `lh-table__multicolumn`? That was an impl detail of v1's handlebars table.